MantidQt/CustomInterfaces/src/Tomography/TomographyIfaceModel.cpp

```diff
@@ -391,6 +488,44 @@ void TomographyIfaceModel::doSubmitReconstructionJob(
   }
 }
 
+void TomographyIfaceModel::doRunReconstructionJobLocal(
+    const std::string &run, const std::string &allOpts,
+    const std::vector<std::string> &args) {
+
+  // Mantid::Kernel::ConfigService::Instance().launchProcess(run, runArgs);
+  try {
+    Poco::ProcessHandle handle = Poco::Process::launch(run, args);
+    Poco::Process::PID pid = handle.id();
+    Mantid::API::IRemoteJobManager::RemoteJobInfo info;
+    info.id = boost::lexical_cast<std::string>(pid);
+    info.name = "Mantid_Local";
+    if (pid > 0) {
+      info.status = "Starting";
+    } else {
+      info.status = "Exit";
+    }
+    info.cmdLine = run + " " + allOpts;
+    m_jobsStatusLocal.emplace_back(info);
+  } catch (Poco::SystemException &sexc) {
+    g_log.error() << "Execution failed. Could not run the tool. Error details: "
+                  << std::string(sexc.what());
+    Mantid::API::IRemoteJobManager::RemoteJobInfo info;
+    info.id = "none";
+    info.name = "Mantid_Local";
+    info.status = "Exit";
+    info.cmdLine = run + " " + allOpts;
+    m_jobsStatusLocal.emplace_back(info);
+  } catch (std::runtime_error &rexc) {
```

Comment:
Can we reduce the catch scope on these at all - I would be surprised if all these steps had the potential to throw